### PR TITLE
iptables intro - closes #67

### LINKED ISSUE #67: IPTables Topic
Create an iptables topic, with an introduction and installation guide

### PR DESCRIPTION
**Pull request by bob-crutchley**

### CODE REVIEW

File: topics/iptables/modules/introduction/README.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+	- [View Current Rules](#view-current-rules)
+- [Allow for SSH Connections](#allow-for-ssh-connections)
+	- [Default Polices](#default-polices)
+	- [Saving](#saving)
+
+<!--TOC_END-->
+## Overview
+Iptables is a command line utility for configuring Linux kernel firewall.
+Iptables is used to inspect, modify, forward, redirect, and/or drop IP packets.
+The code for filtering IP packets is already built into the kernel and is organized into a collection of tables, each with a specific purpose.
```
Comment (tvaidotas) [inline topics/iptables/modules/introduction/README.md]: What is an Ip packet?

File: topics/iptables/modules/introduction/README.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+```bash
+sudo apt install -y iptables
+``` 
+### CentOS/RHEL
+```bash
+sudo yum install -y iptables
+```
+## Tasks
+Here we'll setup a basic chain for using SSH on a machine.
+Be very careful when following the instructions; if you are using a remote machine over SSH for this, there is a chance you can block yourself out if the commands below aren't entered correctly.
```
Comment (tvaidotas) [inline topics/iptables/modules/introduction/README.md]: "if you are using a remote machine over SSH for this, there is a chance you can block yourself out" make this bold

File: topics/iptables/modules/introduction/README.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+In the unlikely case you do get locked out of the machine, you can just restart it to gain access again.
```
Comment (tvaidotas) [inline topics/iptables/modules/introduction/README.md]: Does this mean the rules will be reset after each restart? What's the point of them then

File: topics/iptables/modules/introduction/README.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+   86  7352 ACCEPT     tcp  --  any    any     anywhere             anywhere             tcp dpt:ssh
+
+Chain FORWARD (policy ACCEPT 0 packets, 0 bytes)
+ pkts bytes target     prot opt in     out     source               destination
+
+Chain OUTPUT (policy ACCEPT 44 packets, 5184 bytes)
+ pkts bytes target     prot opt in     out     source               destination
+```
+### Default Polices
+It's best practice to have default rules in place to deny all traffic; we can then create rules with a higher priority to allow certain connections.
```
Comment (tvaidotas) [inline topics/iptables/modules/introduction/README.md]: You mention higher priority only here, but there's nothing else about how to use it, could you add a section about the higher priority and give an example of using it after blocking all with the default rules

File: topics/iptables/modules/introduction/README.md [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+    0     0 ACCEPT     all  --  lo     any     anywhere             anywhere            
+
+Chain FORWARD (policy DROP 0 packets, 0 bytes)
+ pkts bytes target     prot opt in     out     source               destination         
+
+Chain OUTPUT (policy ACCEPT 34 packets, 5832 bytes)
+ pkts bytes target     prot opt in     out     source               destination         
+    0     0 ACCEPT     all  --  any    lo      anywhere             anywhere 
+```
+### Saving
```
Comment (tvaidotas) [inline topics/iptables/modules/introduction/README.md]: Mention that if you don't save it resets in my previous comment part too as it was confusing what's the point of this then

File: topics/iptables/modules/introduction/README.md (hunk 1 above)
Comment (bob-crutchley) [reply topics/iptables/modules/introduction/README.md]: Added a sentence for this

File: topics/iptables/modules/introduction/README.md (hunk 2 above)
Comment (bob-crutchley) [reply topics/iptables/modules/introduction/README.md]: done

File: topics/iptables/modules/introduction/README.md (hunk 3 above)
Comment (bob-crutchley) [reply topics/iptables/modules/introduction/README.md]: Added something in bold to clarify this a bit better

File: topics/iptables/modules/introduction/README.md (hunk 5 above)
Comment (bob-crutchley) [reply topics/iptables/modules/introduction/README.md]: I think there is something in this paragraph already saying that the changes would be lost on a reboot if they haven't been saved; is that what you mean?
```text
If you don't save then the changes made would be erased on a system reboot (which is good if you locked yourself out).
```

File: topics/iptables/modules/introduction/README.md (hunk 4 above)
Comment (bob-crutchley) [reply topics/iptables/modules/introduction/README.md]: There are some commands below that are doing this with the `-P` option; I've included a sentence now to clarify that a bit better

**Pull request merged**